### PR TITLE
Tanzania south 2019 model 

### PR DESCRIPTION
**Pull request by adebowaledaniel**
Trained the model according to issue https://github.com/nasaharvest/crop-mask/issues/214

### CODE REVIEW

File: datasets.py [hunk 1]
```diff
@@ -41,657 +40,77 @@ def clean_ceo_data(df: pd.DataFrame) -> pd.DataFrame:
 
 
 datasets = [
-    CustomLabeledDataset(
```
Comment (ivanzvonkov) [inline datasets.py:44]: If you keep `openmapflow` at or below 0.1.5, these changes are not necessary for now.
Updating to new version can be done separately

File: data/models.json [hunk 2]
```diff
@@ -100,5 +100,22 @@
             "recall_score": 0.6746,
             "roc_auc_score": 0.9222
         }
+    },
+    "Tanzania_south_2019": {
+        "params": "",
```
Comment (ivanzvonkov) [inline data/models.json:105]: Could you train with wandb enabled to allow logging these parameters?

File: notebooks/crop-mask [hunk 3]
```diff
@@ -0,0 +1,326 @@
+{
```
Comment (ivanzvonkov) [inline notebooks/crop-mask:1]: What's this notebook doing here?

File: data/models.json (hunk 2 above)
Comment (adebowaledaniel) [reply data/models.json:105]: I tried adding `--wandb`  parameter, but there is some sort of permission needed.

File: notebooks/crop-mask (hunk 3 above)
Comment (adebowaledaniel) [reply notebooks/crop-mask:1]: Thanks for the observation; this is a mistake.

File: datasets.py (hunk 1 above)
Comment (adebowaledaniel) [reply datasets.py:44]: So should I revert this to the previous version (within the PR)?

**Review by ivanzvonkov — APPROVED** [review]
LGTM

**Pull request merged**